src/ui/interface.js

```diff
@@ -1547,192 +1548,211 @@ export class UI {
 	}
 
 	gridSelectUp() {
-		let game = this.game,
```

Comment:
All the changes from here down are simply renaming variables and reformatting so I could make sense of the code.